src/Symfony/Component/Console/Tests/Formatter/OutputFormatterTest.php

```diff
@@ -189,6 +189,7 @@ public function provideInlineStyleOptionsCases()
             ['<fg=green;options=reverse;>', "\033[32;7m<a>\033[39;27m", '<a>'],
             ['<fg=green;options=bold,underscore>', "\033[32;1;4mz\033[39;22;24m", 'z'],
             ['<fg=green;options=bold,underscore,reverse;>', "\033[32;1;4;7md\033[39;22;24;27m", 'd'],
+            ['<fg=#00ff00;bg=#00f>', "\033[38;2;0;255;0;48;2;0;0;255m[test]\033[39;49m", '[test]'],
```

Comment:
wouldn't this test fail if you run it on a system not supporting true colors ?

Reply:
fixed